components/hackathon-grant/apply2.js

```diff
@@ -0,0 +1,314 @@
+      }}
+    >
+      <Icon glyph={icon} size={48} />
+    </Box>
+    <Box>
+      <Image src="/hackathon-grant/first.png" width="200" height="200" />
+      <Heading as="h3" variant="headline" mb={2}>
+        {name}
+      </Heading>
+      <Text
+        as="p"
+        variant="subtitle"
+        sx={{ mt: 0, pb: 2, a: { variant: 'styles.a', color: 'blue' } }}
+      >
+        {desc}
+      </Text>
+    </Box>
+  </Box>
+)
+
+const Apply = ({ channel }) => {
+  return (
+    <>
+      <Box id="apply" sx={{ pt: 6 }}>
+          <Zoom>
+          <Heading sx={{ textAlign: 'center', mb: 3, fontSize: [5, null, 6, 7] }}>
+          The <MSparkles>bucks</MSparkles> start here.
+        </Heading>
+          </Zoom>
+          <Zoom>
+          <Heading
+          sx={{
+            textAlign: 'center',
+            mb: [2, null, 4],
+            fontSize: [28, 30, 40],
+            color: 'muted'
+          }}
+        >
+          Get your hackathon funded.
+        </Heading>
+          </Zoom>
+      </Box>
+      <Grid
+        pt={[3, 4]}
+        pb={[4, 5]}
+        gap={[4, 3, 4]}
+        columns={[1, 1, 3]}
+        sx={{
+          textAlign: 'left',
+          '> a, > div': {
+            borderRadius: 'extra',
+            boxShadow: 'elevated',
+          },
+          span: {
+            boxShadow:
+              '-2px -2px 6px rgba(255,255,255,0.125), inset 2px 2px 6px rgba(0,0,0,0.1), 2px 2px 8px rgba(0,0,0,0.0625)'
+          },
+          svg: { fill: 'currentColor' }
+        }}
+      >
+          <Slide left delay={20}>
+        <Photo
+          src="/hackathon-grant/step1.gif"
+          alt="Summer Creek Hack Club meeting, February 2020"
+          width={3000}
+          height={2550}
+          showAlt
+          color="#ec3750"
+          icon="slack"
+          duration="Step 1"
+          name={
+            <>
+              On{' '}
+              <Link
+                href="https://hackclub.com/slack"
+                target="_blank"
+                sx={{
+                  color: 'white'
+                }}
+              >
+                Slack
+              </Link>
+              , send your website to{' '}
+              <Link href={channel} target="_blank">
+                #hackathon-grants
+              </Link>
+            </>
+          }
+        />
+        </Slide>
+        <Slide left delay={10}>
+        <Photo
+          src="/hackathon-grant/step2.png"
+          alt="Summer Creek Hack Club meeting, February 2020"
+          width={3000}
+          height={2550}
+          showAlt
+          color="#ec3750"
+          icon="post"
+          duration="Step 2"
+          name={
+            <>
+              Fill in your application, we'll respond in 24 hours
+            </>
+          }
+        />
+        </Slide>
+        <Slide left>
+        <Photo
+          src="/hackathon-grant/step3.png"
```

Comment:
Should the screenshot include the new transaction line item styling?

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/20099646/185644377-bca26a7f-a396-4992-8609-4c9741c0cb75.png">